iocage_lib/ioc_json.py

```diff
@@ -1884,6 +1889,7 @@ def json_check_prop(self, key, value, conf):
             "mount_linprocfs": ("0", "1"),
             "vnet": ("off", "on"),
             "vnet_default_interface": ("string",),
+            "netgraph": ("off", "on"),
```

Comment:
Bool PR change requires this
```suggestion
            "netgraph": truth_variations,
```